frontend/src/fetchers.ts

```diff
@@ -245,10 +245,6 @@ export async function fetchSearchQuery(
   )
 }
 
-export async function fetchLoginProviders() {
-  return axios.get<LoginProvider[]>(LOGIN_PROVIDERS_URL)
```

Comment:
Removing `LOGIN_PROVIDERS_URL` seems to be missing?

Reply:
It's still currently used in the rest of the auth flow.